prepare_data/prepdata/main.py

```diff
@@ -14,10 +14,10 @@ def remove_slice(fname_im, fname_seg):
     """
     # Load data
     nii_im = nib.load(fname_im)
-    data_im = nii_im.get_data()
+    data_im = nii_im.get_fdata()
```

Comment:
[nipy says](https://nipy.org/nibabel/reference/nibabel.dataobj_images.html#nibabel.dataobj_images.DataobjImage.get_data)

> get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

so maybe this should be

```suggestion
    data_im = np.asanyarray(nii_im.dataobj)
```

And I believe this should do the same thing:

```suggestion
    data_im = nii_im.dataobj.get_unscaled()
```

I suppose for many applications it's helpful to just force the data type to float64 for reliability, but for us, since we're always using Nifti1 images which know their data types, we want to avoid the extra work rescaling the data and possibly losing fidelity in the process.

Reply:
as mentioned in [this comment](https://github.com/neuropoly/ivado-medical-imaging/pull/184#discussion_r408997399), `dataobj.get_unscaled()` is not defined (it only works for another type of object: dataoobj (notice the added "o")